port/win/env_win.cc

```diff
@@ -586,16 +585,20 @@ Status WinEnvIO::GetChildren(const std::string& dir,
 
   UniqueFindClosePtr fc(handle, FindCloseFunc);
 
-  if (result->capacity() > 0) {
-    output.reserve(result->capacity());
-  }
-
   // For safety
   data.cFileName[MAX_PATH - 1] = 0;
 
   while (true) {
-    auto x = RX_FILESTRING(data.cFileName, RX_FNLEN(data.cFileName));
-    output.emplace_back(FN_TO_RX(x));
+    // filter out '.' and '..' directory entries
+    // which appear only on some platforms
+    const bool ignore =
+        data.dwFileAttributes & FILE_ATTRIBUTE_DIRECTORY &&
```

Comment:
Can you change this to 
   ((data.dwFiileAttributes & FILE_ATTRIBUTE) != 0) &&
I think it reads a little clearer.

Reply:
Done.